Makefile

```diff
@@ -707,8 +707,13 @@ $(ROM): $(ELF)
 	$(ELF2ROM) -cic 6105 $< $@
 
 $(ROMC): $(ROM) $(ELF) $(BUILD_DIR)/compress_ranges.txt
-	$(PYTHON) tools/compress.py --in $(ROM) --out $@ --dmadata-start `./tools/dmadata_start.sh $(NM) $(ELF)` --compress `cat $(BUILD_DIR)/compress_ranges.txt` --threads $(N_THREADS)
+ifeq ($(PLATFORM),IQUE)
+	$(PYTHON) tools/compress.py --in $(ROM) --out $@ --dmadata-start `./tools/dmadata_start.sh $(NM) $(ELF)` --compress `cat $(BUILD_DIR)/compress_ranges.txt` --format gzip --threads $(N_THREADS)
+	$(PYTHON) -m ipl3checksum sum --cic 6102 --update $@
+else
+	$(PYTHON) tools/compress.py --in $(ROM) --out $@ --dmadata-start `./tools/dmadata_start.sh $(NM) $(ELF)` --compress `cat $(BUILD_DIR)/compress_ranges.txt` --format yaz0 --pad-rom --threads $(N_THREADS)
 	$(PYTHON) -m ipl3checksum sum --cic 6105 --update $@
+endif
```

Comment:
I think something like this would be better
```
ifeq ($(PLATFORM),IQUE)
	COMPRESS_FORMAT := gzip
	CIC := 6102
	EXTRA_ARGS :=
else
	COMPRESS_FORMAT := yaz0
	CIC := 6105
	EXTRA_ARGS := --pad-rom
endif

$(PYTHON) tools/compress.py --in $(ROM) --out $@ --dmadata-start `./tools/dmadata_start.sh $(NM) $(ELF)` --compress `cat $(BUILD_DIR)/compress_ranges.txt` --format $(COMPRESS_FORMAT) $(EXTRA_ARGS) --threads $(N_THREADS)
$(PYTHON) -m ipl3checksum sum --cic $(CIC) --update $@
```